.travis.yml

```diff
@@ -19,6 +19,11 @@ matrix:
       env:
         - OS_TYPE=doxygen
       addons: { apt: { packages: ["cmake", "graphviz"] } }
+    - os: linux
+      dist: xenial
+      env:
+        - OS_TYPE=clangformat
+      addons: { apt: { packages: ["clang-format-6.0"], sources: ["llvm-toolchain-trusty-6.0"] } }
```

Comment:
`llvm-toolchain-trusty-6.0` ?
curios: is this correct?
it's a xenial image

Reply:
`dist:xenial` was added by mistake, in another place I used the same addons but the `dist` was forced to `trusty` for the whole matrix. Fixed the issue.